aws-efa-k8s-device-plugin/templates/daemonset.yaml

```diff
@@ -15,7 +15,6 @@ spec:
       # This annotation is deprecated. Kept here for backward compatibility
       # See https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/
       annotations:
-        scheduler.alpha.kubernetes.io/critical-pod: ""
         {{- with .Values.additionalPodAnnotations }}
```

Comment:
Since these are empty by default, we could have an empty `annotations` block. This should be changed to the same we have for `with .Values.additionalPodLabels` below

Reply:
I may be misunderstanding but annotations is 
```
{{- with .Values.additionalPodAnnotations }}
 {{- toYaml . | nindent 8 }}
{{- end }}
```
and labels (without `name`) is
```
{{- with .Values.additionalPodLabels }}
 {{- toYaml . | nindent 8 }}
{{- end }}
```

The `labels` and `annotations` block are already similar, right?